docs/source/pages/plotting.rst

```diff
@@ -0,0 +1,254 @@
+* ``.plot`` is called on a single returned value by the metric, for example from ``metric.forward()``
+
+In both cases it will generate a plot like this (Accuracy as an example):
+
+.. code-block:: python
+
+    metric = torchmetrics.Accuracy(task="binary")
+    for _ in range(num_updates):
+        metric.update(torch.rand(10,), torch.randint(2, (10,)))
+    fig, ax = metric.plot()
+
+.. plot:: pyplots/binary_accuracy.py
+    :scale: 100
+    :include-source: false
+
+A single point plot is not that informative in itself, but if available we will try to include additional information
+such as the lower and upper bounds the particular metric can take an if the metric should be minimized or maximized
+to be optimal. This is true for all metrics that return a scalar tensor.
+Some metrics returns multiple values (such as an tensor with multiple elements or an dict of scalar tensors), and in
+that case calling ``.plot`` will return a figure similar to this:
+
+.. code-block:: python
+
+    metric = torchmetrics.Accuracy(task="multiclass", num_classes=3, average=None)
+    for _ in range(num_updates):
+        metric.update(torch.randint(3, (10,)), torch.randint(3, (10,)))
+    fig, ax = metric.plot()
+
+.. plot:: pyplots/multiclass_accuracy.py
+    :scale: 100
+    :include-source: false
+
+Here, each element is assumed to be an independent metric and plotted as its own point for comparing. The above is true
+for all metrics that returns a scalar tensor, but if the metric returns a tensor with multiple elements then the
+``.plot`` method will return a specialized plot for that particular metric. Take for example the ``ConfusionMatrix``
+metric:
+
+.. code-block:: python
+
+    metric = torchmetrics.ConfusionMatrix(task="multiclass", num_classes=3)
+    for _ in range(num_updates):
+        metric.update(torch.randint(3, (10,)), torch.randint(3, (10,)))
+    fig, ax = metric.plot()
+
+.. plot:: pyplots//confusion_matrix.py
+    :scale: 100
+    :include-source: false
+
+If you prefer to use the functional interface of Torchmetrics, you can also plot the values returned by the functional.
+However, you would still need to initialize the corresponding metric class to get the information about the metric:
+
+.. code-block:: python
+
+    plot_class = torchmetrics.Accuracy(task="multiclass", num_classes=3)
+    value = torchmetrics.functional.accuracy(
+        torch.randint(3, (10,)), torch.randint(3, (10,)), num_classes=3
+    )
+    fig, ax = plot_class.plot(value)
+
+********************
+Plotting multi steps
+********************
+
+In the above examples we have only plotted a single step/single value, but it is also possible to plot multiple steps
+from the same metric. This is often the case when training a machine learning model, where you are tracking one or
+metrics that you want to plot as they are changing over time. This can be done by provided a sequence of outputs from
```

Comment:
```suggestion
from the same metric. This is often the case when training a machine learning model, where you are tracking one or
multiple metrics that you want to plot as they are changing over time. This can be done by providing a sequence of outputs from
```
was the "multiple" missing?